features/FEATURE_BLE/targets/TARGET_CORDIO/stack/ble-host/sources/sec/common/sec_main.c

```diff
@@ -31,7 +31,7 @@
 #include "hci_api.h"
```

Comment:
out of curiosity - hci_ is not using relative paths like util or pal, why?

Reply:
Because it's not part of pal. This is an external codebase. This is not a valid long term solution and is a stopgap until we fix our build system.

Reply:
This problem has been solved many times and we don't have to develop any cutting edge technology, mbed_lib.json can have a list of include dirs it exports and other libs that require them can import them:
so for cordio host stack it would be "export_include": ["stack/include/wsf/"]
and each library that needed it like cordio controller "depends": ["CORDIO"] which will import the includes that were exported.